homeassistant/components/netgear/sensor.py

```diff
@@ -119,3 +207,60 @@ def async_update_device(self) -> None:
         self._active = self._device["active"]
         if self._device.get(self._attribute) is not None:
             self._state = self._device[self._attribute]
+
+
+class NetgearRouterTrafficEntity(NetgearRouterEntity, SensorEntity):
+    """Representation of a device connected to a Netgear router."""
+
+    _attr_entity_registry_enabled_default = False
+
+    def __init__(
+        self,
+        coordinator: DataUpdateCoordinator,
+        router: NetgearRouter,
+        attribute: str,
+    ) -> None:
+        """Initialize a Netgear device."""
+        super().__init__(coordinator, router)
+        self._attribute = attribute
```

Comment:
This is the same as the entity description key. We can use that instead after we know which description to use.

Reply:
done